### ISSUE TITLE
Legend is squished together

### ISSUE THREAD

**Issue by jjjj12212** [masayuki0812/c3#1005]
I was setting this up today and found the legend is too close together yet my code is I believe is pretty basic. Please Note that the  /head body tags was missing. When I added it, it made no difference.

![2015-02-18_15-23-36](https://cloud.githubusercontent.com/assets/5553022/6259008/b9842342-b782-11e4-8966-ead912ce99cd.png)

![2015-02-18_15-22-07](https://cloud.githubusercontent.com/assets/5553022/6259009/bd7e4932-b782-11e4-91ff-18c93cc271a8.png)

Any ideas how to fix it?

**Comment by ergo** [comment]
Yeah happens also for my charts on random.

**Comment by mrj** [comment]
Commit d35f0f8bfd197a68f729b4511ce1040d5d7573f8 for issue #942 is what broke this.

**Comment by mrj** [comment]
The problem is caused by a function that creates a temporary SVG element to hold legend text so that the dimensions of this text can be determined. The height of this SVG element was set to zero in commit d35f0f8 to prevent the temporary SVG element from flickering the scrollbar (issue #942). This however had the side effect of setting the heights of all legend text boxes to zero.

Perhaps the solution to both problems is to have the temporary SVG element invisible at the top-left of the viewport:
```javascript
svg = body.append("svg").style('visibility', 'hidden').style('position', 'fixed').style('top', 0).style('left', 0)
```

or maybe using absolute you can get away without setting the zero positions:
```javascript
svg = body.append("svg").style('visibility', 'hidden').style('position', 'absolute')
```

**Comment by smuuf** [comment]
@mrj Oh well, thank you! :)

**Comment by masayuki0812** [comment]
Thanks for looking into this. I fixed to use `position = fixed` and now it seems working.
I'll release this fix in the next version 0.4.10 after RC test. Thanks!

**Comment by masayuki0812** [comment]
`0.4.10` has been released, so let me close at the moment. Thanks.
